tests/lib/prepare-restore.sh

```diff
@@ -269,6 +269,8 @@ prepare_project() {
     # no need to modify anything further for autopkgtest
     # we want to run as pristine as possible
     if [ "$SPREAD_BACKEND" = autopkgtest ]; then
+        create_test_user
+        systemctl enable --now snapd.socket
```

Comment:
Debian CI still reports occasional failures. I don't quite know why but I suspect this is only partially sufficient.